### PR TITLE
Using ETC With MetaMask

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
**This post goes on 2022-12-21. When it is reviewed I can publish it at 8 AM EST / 13 HRS UTC as usual.**

This is part of a series: 

- Using ETC With MetaMask
- Using ETC With Trust Wallet
- Using ETC With Ledger Nano S
- Using ETC With MyCrypto
- Using ETC With MyEtherWallet

EN version is ready.

CN version will be pasted as soon as we get the translation.

### CODE REVIEW

**Comment by DonaldMcIntyre** [comment]
> @DonaldMcIntyre you may also consider including Brave Browser in this series? It supports ETC, can easily add via chianlist.org

Thank you for the suggestion. I will add this to the "Idea Bucket" in my editorial calendar.

File: content/blog/2022-12-21-using-ethereum-classic-with-metamask/index.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+It is imperative that you back up your MetaMask wallet because if you lose your phone or your data you will lose access to all your crypto.
+
+To do this, MetaMask has a 12 word secret passphrase that is like a password that may be used to regenerate all your addresses. Once you back it up, you can import your 12 words again in a new device and recover your assets.
+
+Your backup should be in paper and not in digital form because if you store your words in a computer or phone, they may be lost, compromised, or hacked.
+
+![Back up your 12 word secret passphrase and MetaMask password on paper.](./etc-metamask-back-up.png)
+
+Once you have it in paper, you should make a couple of copies and keep them in safe places.
```
Comment (TheCrowbill) [inline content/blog/2022-12-21-using-ethereum-classic-with-metamask/index.md:125]:
```suggestion
Once you have it on paper, you should make a couple of copies and keep them in safe places.
```

**Review by TheCrowbill — APPROVED** [review]
In the section beginning on line 105 regarding using ETH addresses on ETC, I think it would be wise to add the caveat that this is only a certainty if you hold the private keys to the account. If sending to an exchange or custodial wallet, coins are frequently lost when  ETC is sent to an ETH deposit account. At the very least, it is often a hassle to get it all straightened out for an easily avoidable mistake.

File: content/blog/2022-12-21-using-ethereum-classic-with-metamask/index.md (hunk 1 above)
Comment (DonaldMcIntyre) [reply content/blog/2022-12-21-using-ethereum-classic-with-metamask/index.md:125]: I will add your typo and address disclaimer suggestions after publishing. The article approval process is uncertain and I prioritize publishing on time over small typos and the disclaimer is about clarifying that using ETC and ETH addresses interchangeably in general may cause losses, but this post is specifically about MetaMask which means the user does have the private keys of the addresses whether they use them on ETH or ETC.

**Review by w1g0 — COMMENTED** [review]
In the content I would add the tip of best practice by sending a small amount first to a new wallet address before sending the whole amount in risking losing the funds.

**Review by w1g0 — APPROVED** [review]

**Pull request merged**